ui/app/pages/settings/security-tab/security-tab.component.js

```diff
@@ -12,8 +11,6 @@ export default class SecurityTab extends PureComponent {
   }
 
   static propTypes = {
-    setPrivacyMode: PropTypes.func,
-    privacyMode: PropTypes.bool,
     displayWarning: PropTypes.func,
     revealSeedConfirmation: PropTypes.func,
     showClearApprovalModal: PropTypes.func,
```

Comment:
This prop is no longer used - it can be removed

Reply:
Good catch, thanks